evm_loader/program/src/evm/mod.rs

```diff
@@ -255,6 +256,15 @@ impl<B: Database> Machine<B> {
         });
 
         let status = loop {
+            if is_precompile_address(&self.context.contract) {
+                let value =
+                    precompile(&self.context.contract, &self.call_data.clone()).unwrap_or_default();
```

Comment:
You don't need to clone data here

Reply:
@anton-lisanin Correct, `clippy` didn't help me this time, removed clone()